src/Sulu/Bundle/AdminBundle/Resources/views/Admin/main.html.twig

```diff
@@ -12,6 +12,7 @@
         <script type="text/javascript">
             {% autoescape false %}
             const SULU_CONFIG = Object.freeze({
+                initialLoginState: {{ app.user ? 'true' : 'false' }},
```

Comment:
maybe we should use `is_granted('IS_AUTHENTICATED_REMEMBERED')` here instead of checking if `app.user` variable exist as I remeber it could be in some cases be string with the name `.anom`.

Reply:
:+1:

Reply:
done 🙂